addons/payment_mercado_pago/const.py

```diff
@@ -30,7 +30,8 @@
 # Mapping of transaction states to Mercado Pago payment statuses.
 # See https://www.mercadopago.com.mx/developers/en/reference/payments/_payments_id/get.
 TRANSACTION_STATUS_MAPPING = {
-    'pending': ('pending', 'in_process', 'in_mediation'),
+    'pending': ('pending', 'in_process', 'in_mediation', 'authorized'),
```

Comment:
Indeed, `authorized` means that the funds are reserved and the payment method is valid; the ongoing operation should be allowed to continue (e.g. shipping).

Reply:
Hi @maq-adhoc 

> A payment with authorized status can change to status (done, cancel, rejected). It is necessary that the state of the transaction is pending.

An authorized payment can indeed change to `done`, but how could it be canceled or rejected without user input? To my understanding, _automatic_ capture payments are authorized right before they are captured.

Or maybe the user has a way to update the payment from MercadoPago's backend in between the authorization and the capture?

I'm not implying that what you suggest is impossible; I'm just trying to understand how MercadoPago differs from the other providers we have implemented so far :wink: 

I agree with the rest of your statements.

In the end, regardless of whether authorized payments can be canceled or not, I agree to flag them as `pending` because we should always receive a capture webhook notification anyway if/when they are captured. cc @Demesmaeker